src/os_auth/auth.c

```diff
@@ -165,6 +165,9 @@ w_err_t w_auth_parse_data(const char* buf,
             snprintf(ip, IPSIZE, "%s", client_source_ip);
         }
 
+        /* Forward the string pointer to allow the proper key parsing IP:'........'
```

Comment:
This comment should only refer to the I parsing like in the Groups comment. i.e.:
/* Forward the string pointer IP:'........' 3 for IP:, 2 for '' */

Reply:
Fixed!